docs/releases/updating_ez_platform.md

```diff
@@ -261,6 +261,22 @@ If you are upgrading to v1.8 you need to create this folder and assign it to a n
     form_builder.upload_folder.section_identifier: <section identifier>
 ```
 
+**Increased password hash length**
+
+V1.12
```

Comment:
With current doc versioning, we don't need lines like this anymore (they will be removed from the rest of the file soon(TM))

Reply:
Expunged.